Documentation/NextSteps/Editors/Index.rst

```diff
@@ -7,16 +7,11 @@
 For Editors
 ===========
 
-Introductions
-=============
-
 Tutorial for Editors
---------------------
-
-The :ref:`t3editors:start` explains how to add or change content on an existing website in TYPO3 CMS.
-It provides the most basic step-by-step instructions to get the job
-done. This is very likely the next step that you want to take.
+=============
```

Comment:
Underlining should always be at least the same length as title. Otherwise there may be problem with rendering, see https://docs.typo3.org/typo3cms/HowToDocument/WritingReST/CommonPitfalls/Headers.html

Here, the underline is now for the title "Tutorial for Editors". 

I will push another commit to fix.